src/libsystemd/sd-network/sd-network.c

```diff
@@ -270,6 +270,10 @@ int sd_network_link_get_route_domains(int ifindex, char ***ret) {
         return network_link_get_strv(ifindex, "ROUTE_DOMAINS", ret);
 }
 
+int sd_network_link_get_pref64(int ifindex, char ***ret) {
```

Comment:
Please do not expose new option to `sd-network`, but expose through json format. See networkd-json.[ch].